src/04-meet-your-hardware/README.md

```diff
@@ -43,13 +43,15 @@ current flows through a circuit. By enabling or disabling electrical current to
 flow through a specific pin, an LED attached to that pin (via the traces) can
 be turned on and off.
 
-By looking at the part number and doing some research, we can learn more
-information about electronic components. The part number of our microcontroller
-starts with `STM32`, which indicates that it was manufactured by [ST
-Microelectronics]. It also lets us know that this is an Arm®-based 32-bit
-microcontroller. The `F3` in the part number indicates that the microcontroller
-is based on the Cortex®-M4. The rest of the part number provides more specifics
-(e.g., extra features, RAM size), and aren't as important to us right now.
+Part numbers of components can often give us information about the part. In the
+case of our MCU (with part number `STM32F303VCT6`), the `STM` at the start
+hints to use that this is manufactured by [ST Microelectronics]. Each
+manufacturer uses a different part numbering scheme, but with some research we
+can also determine that the `32` implies that this is specifically an
+Arm®-based 32-bit microcontroller, and the `F3` indicates that the
+microcontroller is based on the Cortex®-M4. The rest of the part number
+provides more specifics (e.g., extra features, RAM size), and aren't as
+important to us right now.
```

Comment:
I think it's more accurate to say the `ST` at the start tells us it's made by ST, while the `M32` tells us it's a 32-bit microcontroller, and the `F3` tells us it's in ST's `STM32F3` family of Cortex-M4-based microcontrollers. I concede your point on the link; ST have definitely changed them in the past and it's pretty easily found with a search anyway.

Reply:
Ah, thanks. Changed in 018702e0332ab864e7e4a59b4a2f49bbd3d78669. Let me know if that makes more sense.

Reply:
Thanks! My only remaining quibble is that `F3` by itself doesn't tell us that it's based on a Cortex-M4 core (you might even expect it to be a Cortex-M3 core given the `3`); it tells us it's part of the STM32F3 family and you just have to know that the F3 family uses a Cortex-M4 core.

Reply:
Ah, okay, I see what you're saying. Yeah, I can see how that could be confusing, and why the link would be helpful.

How about something like this:

> Each manufacturer uses a different part numbering scheme. Looking at our MCU's part number (`STM32F303VCT6`), the `ST` at the front hints to us that this is a part manufactured by [ST Microelectronics](https://www.st.com/). Searching through [ST's marketing materials](https://www.st.com/en/microcontrollers-microprocessors/stm32-mainstream-mcus.html), we can also learn the following:
> * The `M32` represents that this is an Arm®-based 32-bit microcontroller.
> * The `F3` represents that the microcontroller is based on the Cortex®-M4 processor design.
> * The remainder of the part number goes into more details about things like extra features and RAM size, which at this point we're less concerned about.

Reply:
Sorry, meant to also say: yes, the link has the chance to break. But as you mentioned, it's easy to search for even if the link breaks.

Reply:
The second link does look better. I guess I'm not sure about saying the `F3` "represents" that it's based on a Cortex-M4, since you can't infer that from the `F3`. Perhaps something like...

> * The M32 means this is one of ST's Arm®-based STM32 32-bit microcontrollers.
> * The F3 means this microcontroller is in ST's STM32F3 series, which use Arm's Cortex®-M4 processor design.

Reply:
Gotcha, yeah I seem to keep skipping that step, huh? Thanks for keeping me honest. I've tried to put this together in 524530728410f10b1231d51b94d9880f0faec115, this time meantioning that F3 describes a series of MCU, and that the series utilizes the Cortex-M4.